packages/terra-core-docs/src/terra-dev-site/doc/alert/example/AlertActionFocusDemo.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import Button from 'terra-button';
+import Alert from 'terra-alert';
+
+const AlertActionFocusDemo = () => {
+  const [actionButtonClickCount, setActionButtonClickCount] = useState(0);
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <>
+      {isOpen && (
+        <Alert
+          onDismiss={() => setIsOpen(false)}
+          action={(
+            <Button
+              text="Action"
+              variant="emphasis"
+              onClick={() => {
+                setActionButtonClickCount((prevCount) => prevCount + 1);
+              }}
+            />
+          )}
+        >
+          The notification banner should be focused upon render when navigating via keyboard.
+        </Alert>
+      )}
+      <Button
+        isDisabled={isOpen}
+        text="Trigger Alert Notification Banner Demo"
+        onClick={() => {
+          setIsOpen(true);
+        }}
+      />
+      <p>{`Action button has been clicked ${actionButtonClickCount} times.`}</p>
```

Comment:
This bit isn't useful for this example in this context.

Reply:
Removed the action button count here https://github.com/cerner/terra-core/pull/3815/commits/37c7002c9b4af70467996103b95af10fb167390d#diff-d14b87cf829844201ff9beefbe8e27f19748a7616848054f1f39051c4ae94a07